src/stream/mod.rs

```diff
@@ -19,6 +19,14 @@ use std::task::{Context, Poll};
 use tokio::io::{AsyncRead, AsyncWrite, ReadBuf};
 use tokio::sync::{mpsc, Mutex, Notify};
 
+/// Possible values which can be passed to the [`Stream::shutdown`] method.
+#[derive(PartialEq)]
```

Comment:
Probably want Debug

Reply:
I actually don't need it. Can just use `std::net::Shutdown`.